### PR TITLE
Asym clean up

### PR DESCRIPTION
**Pull request by mdesco**
- 2 functions out of the script
- new module aodf.py in reconst

Test passed.

### CODE REVIEW

File: scilpy/reconst/aodf.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+
+#
+# aodf := Assymetric Orientation Distribution Function (aODF)
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:5]: asymmetric*

File: scilpy/reconst/aodf.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#
+
+import numpy as np
+
+from dipy.reconst.shm import sph_harm_ind_list
+
+
+def compute_asymmetry_index(sh_coeffs, order, mask):
+    _, l_list = sph_harm_ind_list(order, full_basis=True)
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:14]: docstring:
```
"""
Compute asymmetry index (ASI) (Cetin-Karayumak et al, 2018) from
asymmetric ODF volume expressed in full SH basis.

Parameters
-----------------
sh_coeffs: ndarray (x, y, z, ncoeffs)
    Input spherical harmonics coefficients.
order: int > 0
    Max SH order.
mask: ndarray (x, y, z), bool
    Mask inside which ASI should be computed.

Returns
-----------
asi_map: ndarray (x, y, z)
    Asymmetry index map.
"""
```

File: scilpy/reconst/aodf.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        np.sum(sh_squared, axis=-1)[mask]
+
+    # Negatives should not happen (amplitudes always positive)
+    asi_map = np.clip(asi_map, 0.0, 1.0)
+    asi_map = np.sqrt(1 - asi_map**2) * mask
+
+    return asi_map
+
+
+def compute_odd_power_map(sh_coeffs, order, mask):
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:32]:
```
"""
Compute odd-power map (Poirier et al, 2020) from
asymmetric ODF volume expressed in full SH basis.

Parameters
-----------------
sh_coeffs: ndarray (x, y, z, ncoeffs)
    Input spherical harmonics coefficients.
order: int > 0
    Max SH order.
mask: ndarray (x, y, z), bool
    Mask inside which odd-power map should be computed.

Returns
-----------
odd_power_map: ndarray (x, y, z)
    Odd-power map.
"""
```

File: scilpy/reconst/aodf.py (hunk 1 above)
Comment (CHrlS98) [reply scilpy/reconst/aodf.py:5]: + I think you could remove some empty lines above

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-776/2/

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scilpy/reconst/aodf.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+from dipy.reconst.shm import sph_harm_ind_list
+
+
+def compute_asymmetry_index(sh_coeffs, order, mask):
+    """
+    Compute asymmetry index (ASI) (Cetin-Karayumak et al, 2018) from
+    asymmetric ODF volume expressed in full SH basis.
+
+    Parameters
+    -----------------
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:17]: Make the "--------" fit the length of the word above

File: scilpy/reconst/aodf.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    sh_coeffs: ndarray (x, y, z, ncoeffs)
+         Input spherical harmonics coefficients.
+    order: int > 0
+         Max SH order.
+    mask: ndarray (x, y, z), bool
+         Mask inside which ASI should be computed.
+
+    Returns
+    -----------
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:26]: same

File: scilpy/reconst/aodf.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    return asi_map
+
+
+def compute_odd_power_map(sh_coeffs, order, mask):
+    """
+    Compute odd-power map (Poirier et al, 2020) from
+    asymmetric ODF volume expressed in full SH basis.
+
+    Parameters
+    -----------------
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:55]: same

File: scilpy/reconst/aodf.py [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    sh_coeffs: ndarray (x, y, z, ncoeffs)
+         Input spherical harmonics coefficients.
+    order: int > 0
+         Max SH order.
+    mask: ndarray (x, y, z), bool
+         Mask inside which odd-power map should be computed.
+
+    Returns
+    -----------
```
Comment (CHrlS98) [inline scilpy/reconst/aodf.py:64]: same

**Review by CHrlS98 — COMMENTED** [review]
Just this style thing and then it will be ready to merge

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**